packages/emoji-mart/src/components/Picker/Picker.js

```diff
@@ -936,13 +953,19 @@ export default class Picker extends Component {
         class="flex flex-auto flex-center flex-middle"
         style={{
           position: 'relative',
-          width: this.props.emojiButtonSize,
-          height: this.props.emojiButtonSize,
+          width: this.props.skinToneLabel ? 'auto' : this.props.emojiButtonSize,
+          height: this.props.skinToneLabel
+            ? 'auto'
+            : this.props.emojiButtonSize,
         }}
       >
+        {this.props.skinToneLabel && (
+          <span class="skin-tone-button-label">{this.props.skinToneLabel}</span>
+        )}
         <button
           type="button"
           ref={this.refs.skinToneButton}
+          part="skin-tone-button"
```

Comment:
It seems like with the shadow DOM, `part` is needed to be able to apply styles to anything? We desire a greater amount of control over how this button looks but because `emoji-mart` is now in the shadow DOM it's difficult to style.